composer.json

```diff
@@ -21,12 +21,13 @@
     "symfony/framework-bundle": "^3.4 || ^4.4"
   },
   "require-dev": {
-    "ekino/phpstan-banned-code":   "^0.2",
-    "friendsofphp/php-cs-fixer":   "^2.16",
-    "phpstan/phpstan-phpunit":     "^0.11",
-    "phpunit/phpunit":             "^7.2",
+    "dg/bypass-finals": "^1.2",
```

Comment:
Maybe we could just use the `@final` annotation. Is has the same semantic value than a real final and avoid extra code to be able to mock. A real final and `@final` is handled the same by Symfony's DebugClassLoader for example.

Reply:
But @final annotation does not secure the code against further extension, it is just for documentation purpose, am I right ?